packages/salesforcedx-utils-vscode/src/context/workspaceContextUtil.ts

```diff
@@ -52,7 +56,7 @@ export class WorkspaceContextUtil {
   }
 
   public static getLogDirPath(): string {
-    return join(getRootWorkspacePath(), '.sfdx', 'tools', 'debug', 'logs');
+    return join(workspaceUtils.getRootWorkspacePath(), SFDX_FOLDER, DEBUG_LOGS);
```

Comment:
Let's try replacing the usages of this function with the new one from projectPaths.  Like:

<img width="400" alt="2022-10-27_12-10-55" src="https://user-images.githubusercontent.com/46458081/198378072-ba7f7a04-36e6-4587-8d33-2628ea81fc42.png">

Then we might be able to remove this "getLogDirPath()" function entirely